test/e2e/stargate_apis_test.go

```diff
@@ -33,30 +33,18 @@ func testStargateApis(t *testing.T, ctx context.Context, k8sContextName string,
 			t.Log("test Stargate REST API in context " + k8sContextName)
 			testStargateRestApis(t, k8sContextIdx, username, password, replication)
 		})
-		if t.Failed() {
-			for _, rack := range stargateRacks {
-				dumpStargateLogs(k8sContextName, namespace, dcName, rack)
-			}
-		}
 	})
 }
 
-func dumpStargateLogs(k8sContextName string, namespace string, dc string, rack string) {
-	deploymentName := fmt.Sprintf("deployment.apps/test-%s-%s-stargate-deployment", dc, rack)
-	cmd := exec.Command("kubectl", "logs", deploymentName, "-n", namespace, "--context", k8sContextName)
-	output, _ := cmd.CombinedOutput()
-	println()
-	fmt.Println("=============", "BEGIN STARGATE LOGS", dc, rack, "context", k8sContextName, "namespace", namespace, "=============")
-	fmt.Println(string(output))
-	fmt.Println("=============", "END STARGATE LOGS", dc, rack, "context", k8sContextName, "namespace", namespace, "=============")
-	println()
-}
-
 func testStargateRestApis(t *testing.T, k8sContextIdx int, username string, password string, replication map[string]int) {
 	restClient := resty.New()
 	token := authenticate(t, restClient, k8sContextIdx, username, password)
```

Comment:
Do we need to authenticate again here? Could we reuse the token acquired in `DeployStargateIngresses`? I am mostly asking out of curiosity and don't want to change anything at the moment unless absolutely necessary.